packages/inter-protocol/src/vaultFactory/vault.js

```diff
@@ -766,22 +768,43 @@ const selfBehavior = {
    *
    * @see getActualDebAmount
    *
-   * @param {MethodContext} context
    * @returns {import('./storeUtils.js').NormalizedDebt} as if the vault was open at the launch of this manager, before any interest accrued
    */
-  getNormalizedDebt: ({ state }) => {
+  getNormalizedDebt() {
+    const { state } = this;
     // @ts-expect-error cast
     return reverseInterest(state.debtSnapshot, state.interestSnapshot);
   },
 };
 
-const makeVaultBase = defineDurableKindMulti(
-  makeKindHandle('Vault'),
-  initState,
+export const VaultI = M.interface('Vault', {
+  getCollateralAmount: M.call().returns(AmountShape),
+  getCurrentDebt: M.call().returns(AmountShape),
+  getNormalizedDebt: M.call().returns(AmountShape),
+  getVaultSeat: M.call().returns(SeatShape),
+  initVaultKit: M.call(SeatShape, M.any()).returns(M.promise()),
+  liquidated: M.call().returns(undefined),
+  liquidating: M.call().returns(undefined),
+  makeAdjustBalancesInvitation: M.call().returns(M.promise()),
+  makeCloseInvitation: M.call().returns(M.promise()),
+  makeTransferInvitation: M.call().returns(M.promise()),
+});
+
+const vaultKind = makeKindHandle('Vault');
+const makeVaultBase = defineDurableFarClassKit(
+  vaultKind,
   {
-    self: selfBehavior,
-    helper: helperBehavior,
+    helper: M.interface(
+      'DepositFacet',
+      {
+        // helper not exposed so guard not necessary
```

Comment:
I started writing these out and it felt like burning time. Before we had far classes these functions would just be in closure scope, never exposed. We didn't write validation for the params and returns of all those helpers. I don't think you're proposing that we do so for all helper functions, and the fact that it's in a far class doesn't change the need for protection from mistakes.

I'd be very interested to see an attack that passed the external interfaces but succeeded because some helper function didn't have a guard on its params and/or return. If that is demonstrated, then I think we should consider removing the `UnguardedHelperI` _and_ guarding our other helper functions too.